options/rtdl/aarch64/elf.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <elf.h>
+
+#define ELF_CLASS 64
+
+using elf_ehdr = Elf64_Ehdr;
+using elf_phdr = Elf64_Phdr;
+using elf_dyn = Elf64_Dyn;
+using elf_rel = Elf64_Rel;
+using elf_rela = Elf64_Rela;
+using elf_sym = Elf64_Sym;
+using elf_addr = Elf64_Addr;
+
+using elf_info = Elf64_Xword;
+using elf_addend = Elf64_Sxword;
```

Comment:
Would making the code generic in this manner not prevent a 64-bit loader from loading 32-bit executables? Or is this not something we support at all?

Reply:
This is equivalent to the macro magic in place with other libc implementations. For loading 32-bit executables on e.g. x86_64, I'd assume the 32-bit ld.so would be requested.